src/test/scala/biowdl/test/TestCompareGffTest.scala

```diff
@@ -23,7 +23,4 @@ package biowdl.test
 
 import nl.biopet.utils.biowdl.samples.{Wgs1PairedEnd, Wgs2PairedEnd}
 
-class TestPipelineTest
-    extends TestPipelineSuccess
-    with Wgs1PairedEnd
-    with Wgs2PairedEnd
+class TestCompareGffTest extends TestCompareGffSuccess
```

Comment:
This should extend `TestAnnotation` and define the `sampleGtf`.
Also, `Test` is in the class name twice now, which is kind of redundant.